src/backends/torch/torchlib.cc

```diff
@@ -688,7 +688,7 @@ namespace dd
             if (!ad_distort.empty())
               {
                 distort_params._prob = ad_distort.get("prob").get<double>();
-                this->_logger->info("noise: {}", distort_params._prob);
+                this->_logger->info("distort: {}", distort_params._prob);
```

Comment:
This should have been already updated on master.

Reply:
[It's not](https://github.com/jolibrain/deepdetect/blob/a03eb87fcd60267deac403e33850fd38c6a7760e/src/backends/torch/torchlib.cc#L691)

Reply:
ok